scripts/convert_merged_tract_to_dpdd.py

```diff
@@ -21,6 +21,7 @@
 pyarrow or fastparquet.
 """
 
+import os
```

Comment:
[minor] You could save one import by using only

```python
import os
```
And then using `os.sys.<>` when necessary (only once for the moment, when parsing).